### DISCUSSION TITLE
I recently changed my username twice already and github pages still doesn't work for me, Could someone explain what I'm doing wrong?

### DISCUSSION THREAD

**Discussion by realzoeygreen** [community/community#40128] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've been trying to make a github pages website but it doesn't work, I don't know why :<

**Comment by DSDmark** [comment, marked as answer]
Hi @zackgreen16,

in which repository you are facing that problem. You can your repo link.

And also, Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **error** which you were facing 👍.

**Reply by realzoeygreen** [reply]
Repo link: https://github.com/zackgreen16/zackgreen16.github.io

So what's happening is that the github pages option isn't appearing even though I made the github repo name a github pages link like it should be
![image](https://user-images.githubusercontent.com/66157074/204085047-805b25d3-37bd-44c7-9b7a-856a9d5d25a8.png)

**Reply by DSDmark** [reply]
Hi @zackgreen16,

The above link which you provided is link of **empty** repository. So, even if you set up **GitHub action,** you can't manage you deploy your **github-page**. If sound like complication then ignore all this and remember one thing first you need **project** which contain your website ok. after that.

- Go to setting and click on page section. You can deploy your page there like this.

![2022-11-26_17-35](https://user-images.githubusercontent.com/85744816/204086920-e3973978-e9aa-448f-8268-22af6a88d3e9.png)

here, check this out for more details :- (https://github.com/orgs/community/discussions/38632#discussioncomment-4092967)

**Reply by DSDmark** [reply]
You Can't see github-page in there. As What you are talking in above image, that was the older version of GitHub supported

**Reply by realzoeygreen** [reply]
Oh alright, Sorry for the misunderstanding
I’ve been watching way too many Youtube videos showing the old way and the old way worked for my friend who made a GitHub account yesterday for his website

I’ll mark this as the answer right now, Thank you Sanskar Sahu!
